travis-ci/terraform-config
# Enable build tracing for MacStadium workers

## What is the problem that this PR is trying to fix?

Worker needs various configuration to enable build tracing, and we want to support that in MacStadium like the other infras.

## What approach did you choose and why?

Create S3 users org and com, staging and production. Use these to add appropriate configuration to the workers.

## How can you test this?

Deploy it and confirm in staging that build traces are being created when enabled for the repo.

## What feedback would you like, if any?

* Do we need to also turn this on for the custom workers?
  * If so, do they need their own S3 users/buckets or can they just go in .com?
  * If not, is it an issue that the common configuration for all workers turns on build tracing, but the custom workers don't have credentials or bucket configuration?
* Did I miss anything important here?
